test/suites/fips/validate-fips.robot

```diff
@@ -0,0 +1,52 @@
+*** Settings ***
+Documentation       Tests related to FIPS Validation
+
+Resource            ../../resources/ostree-health.resource
+Resource            ../../resources/common.resource
+Resource            ../../resources/selinux.resource
+Library             Collections
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+
+*** Variables ***
+${USHIFT_HOST}      ${EMPTY}
+${USHIFT_USER}      ${EMPTY}
+
+
+*** Test Cases ***
+Verify Host Is FIPS Enabled
+    [Documentation]    Performs a FIPS validation against the host
+    Wait Until Greenboot Health Check Exited
+    Fips Should Be Enabled
+
+Verify Binary Is FIPS Compliant
+    [Documentation]    Performs a FIPS validation against the Microshift binary
+    Microshift Binary Should Dynamicly Link FIPS Ossl Module
```

Comment:
Is there a limit in the number of characters in a keyword? Asking because of the `Dynamicly`

Reply:
Yes, we might want to fix the spelling. Good catch!

Reply:
fixed the spelling .. not sure about such limitation